db/migrate/20171009230347_create_resources.rb

```diff
@@ -1,7 +1,7 @@
 class CreateResources < ActiveRecord::Migration[5.1]
   def change
     create_table :resources do |t|
-      t.string :file_name
+      t.string :fileName
```

Comment:
can you revert this change? i did a hotfix for this last night so when you merge with master the issue we were having should be fixed already.